selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -2290,6 +2290,26 @@ public void checkProposalDocumentation(String expectedText) {
     seleniumWebDriverHelper.waitTextContains(proposalDoc, expectedText);
   }
 
+  /** enter the 'Ctrl + F12' */
```

Comment:
Looks redundant, IMHO.

Reply:
You mean, that may skip the comment line?

Reply:
I meant that we can use `seleniumWebDriverHelper.press...` method directly.